exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -97,6 +102,24 @@ export function getResumedPackageChunks ({
   return chunks.slice(chunkPosition)
 }
 
+export async function writeRecursiveSummary (opts: { dir: string, summary: RecursiveSummary, reportSummary?: boolean }) {
+  if (opts.reportSummary) {
+    await writeJsonFile(path.join(opts.dir, 'pnpm-exec-summary.json'), opts.summary)
```

Comment:
write it to a field in case we'll need to add other fields in the future.
```suggestion
    await writeJsonFile(path.join(opts.dir, 'pnpm-exec-summary.json'), { executedPackages: opts.summary })
```

Reply:
`executedPackages` feels a bit inaccurate, because it may also contain queued packages, how about naming it `executedStatus`?

Reply:
executionStatus then